samza-core/src/main/java/org/apache/samza/config/ClusterManagerConfig.java

```diff
@@ -88,10 +88,20 @@
   /**
    * Determines whether a JMX server should be started on the job coordinator
    * Default: true
+   *
+   * @deprecated use {@code JOB_JMX_ENABLED} instead
```

Comment:
Reorganize these fields. Put the job.jmx.enabled one on top, move these to below it with a note to use job.jmx.enabled instead, and remove duplicate javadoc.

Reply:
sure